File: src/Example3Java/EX09.java
package Example3Java;

import java.util.Scanner;

public class EX09 {
    public static void main(String[] args) {
        System.out.print("Enter Text : ");
        String revers = "",str = new Scanner(System.in).nextLine();
        for (int i = str.length() - 1; i >= 0; i--) revers = revers + str.charAt(i);
        System.out.println((str.equals(revers)? "It is palindrom" : "It is not palindrom"));
    }
}

File: src/Example3Kotlin/EX09.kt
import java.util.*

object EX09 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter Text : ")
        val str = Scanner(System.`in`).nextLine()
        var revers = ""
        for (i in str.length - 1 downTo 0) revers += str[i]
        println(if (str == revers) "It is palindrom" else "It is not palindrom")
    }
}
